### PR TITLE
'Marketing & Release' Page

### PR DESCRIPTION
**Pull request by arvin-shafiei**
Created a Marketing & Release page based on the theme in #15 wasn't exactly sure what theme we are using for the website since I also noticed a different design in #13 but shouldn't be too hard to change if required

image

### CODE REVIEW

**Comment by ke444a** [comment]
Great job! What do you think about adding some visual components to your pages? Images/videos/slideshows with images or anything else. I guess it would look just more representative.

**Comment by arvin-shafiei** [comment]
82e918d8bab14bbe3391b2aa823b6b27

File: marketing_release.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                        <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
+                            <a class="dropdown-item pdfont" href="marketing_release.html">MARKETING AND RELEASE</a>
+                            <a class="dropdown-item" href="legacy.html">LEGACY</a>
+                            <a class="dropdown-item" href="remaster.html">REMASTER</a>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
```
Comment (ke444a) [inline marketing_release.html:74]: You need to update your header to the new one that our team uses for the website (you could find it in my code on Home Page)

File: marketing_release.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+    <main class="bgimg3">
+        <!-- Main text container-->
+        <div class="container pt-3 px-4 rounded greybg">
```
Comment (ke444a) [inline marketing_release.html:78]: This div is not closed. You need to type additional </div> to close this one

**Review by ke444a — COMMENTED** [review]
The page is well-done, but there's a tiny mistake in the HTML file that I spotted. Also, you need to update the header. After making these changes, the page is ready to be merged.

**Pull request merged**